recipes/archicad-apidevkit/all/conanfile.py

```diff
@@ -0,0 +1,36 @@
+from conan import ConanFile
+from conan.tools.files import get
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.00"
+
+
+class ArchicadApidevkitConan (ConanFile):
+    name = "archicad-apidevkit"
+    description = "The General API Development Kit enables software developers to extend the functionality of Archicad"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://archicadapi.graphisoft.com/"
+    license = "https://archicadapi.graphisoft.com/documentation/license-agreement"
+    settings = "os", "arch"
+    no_copy_source = True
+    topics = "API, Archicad, development"
```

Comment:
```suggestion
    topics = "api", "archicad", "development"
```
Topics is a list and should use lower-case, so the search engine will not mess similar names and using the Conan center page.

Reply:
Sorry Archicad is a brand name which must be spelled with capital letter